mixer/pkg/api/grpcServer.go

```diff
@@ -230,6 +367,83 @@ func (s *grpcServer) check(ctx context.Context, req *mixerpb.CheckRequest,
 	return resp, nil
 }
 
+
+//Access log service shold return empty response
+func (s *grpcServerEnvoy) StreamAccessLogs(
+	srv accessLogGRPC.AccessLogService_StreamAccessLogsServer) (error) {
+	for {
+		ctx := context.TODO()
+
+		msg, err := srv.Recv()
+		if err != nil {
+			return err
```

Comment:
curious: is this streaming grpc convention (return error for any individual message error)?  or should some handling be provided for expected errors, like `EOF` ?

I was sort of expecting something like:

```
 if err == io.EOF {
        break
 }
```

Reply:
done